style.css

```diff
@@ -0,0 +1,112 @@
+@import url('https://fonts.googleapis.com/css2?family=crete+Round&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+}
+
+body {
+  height: 100vh;
+  width: 100%;
+}
+
+header {
+  height: 10vh;
+  min-width: 100%;
+  background-color: #3c3a39;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+nav {
+  width: 100%;
+  margin: 20px;
+  padding: 10px;
+  display: inherit;
+  flex: 1 1;
+}
+
+header nav a {
+  display: flex;
+  color: white;
+  text-decoration: none;
+  font-size: 28px;
+  width: 50%;
+}
+
+header nav .icon {
+  justify-content: flex-end;
+}
+
+main {
+  display: flex;
+  padding: 20px;
+  flex-direction: column;
+  align-items: center;
+  min-height: 86vh;
+  background-color: #1c1a19;
+}
+
+main .text-content {
+  margin: 30px 20px 0 30px;
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+}
```

Comment:
- [ ] DRY stands for `Don't Repeat Yourself`, however you are repeating ourselves in your code, `display:flex , align-items`.

How do you keep CSS dry?

   DRY CSS comes down to 3 things:
- Group reusable css properties together.
- Name these groups logically.
- Add your selectors to the various css groups.

You can make a class and call it `.d-flex` to be called wherever you need that action